engine/execution/state/state.go

```diff
@@ -454,21 +454,17 @@ func (s *state) saveExecutionResults(
 		}
 
 		executionResult := &result.ExecutionReceipt.ExecutionResult
-		err = s.results.BatchStore(executionResult, batch)
```

Comment:
`s.myReceipts.BatchStoreMyReceipt` will store the result, so no need to store the result here.